students/ptfbanks/Lesson03/3-2_list_lab.py

```diff
@@ -0,0 +1,64 @@
+print ("Today we are going to be playing with fruits.  It's a healthy exercise.  Don't take it personally")
+print ("We start with a list of:")
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+print (fruits, "\n")
+add_entry = input("Enter your additional favorite fruit: ")
+fruits.append(add_entry)
+print ("Thank you.  The fruit sequence is now:")
+print(fruits,"\n")
+
+i = int(input("Now, in that list, which is your favoite?, Select by number, 1, 2,... :  "))
+print("Ahh, Good Choice, number", (i), " ", fruits[i-1], "\n")
+
+add_entry = input("Another fruit to lead the list? ")
+fruits = [add_entry] + fruits
+print (fruits,"\n")
+
+add_entry = input("Please add another fruit to lead the list?")
+
+fruits.insert(0,add_entry)
+print (fruits, "\n")
+
+print ("interesting.....  All fruits beginning with 'P' include:")
+p_fruits = []
+for element in fruits:
+    if element[0]== "p":
```

Comment:
Good idea to use lower() here to make it case-insensitive.  i.e., if element[0].lower() == "p":